src/ert/scheduler/openpbs_driver.py

```diff
@@ -303,6 +287,7 @@ async def poll(self) -> None:
                     "qstat",
                     "-fx",
                     "-Fjson",
+                    "-w",  # wide format
```

Comment:
This is not relevant here when we have the json format, it will not be truncated.